Bottom: 811860feef260cde0db289f5835e88c9ac74500b
Top:    8296933bdb360741edb27e420bf5cce15e0b1456
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:20 +0000

Fix undo in MoveObjectAndRubberband

XXX: Need to check if the path which previously missed
     IncrementUndoSerialNumber() was doing so for a reason.


---

diff --git a/src/move.c b/src/move.c
index cfdf881..15cbddc 100644
--- a/src/move.c
+++ b/src/move.c
@@ -946,6 +946,7 @@ MoveObjectAndRubberband (int Type, void *Ptr1, void *Ptr2, void *Ptr3,
       MoveLinePoint (Ptr1, moving_line, &moving_line->Point2);
 
       ptr2 = moving_line;
+      IncrementUndoSerialNumber ();
     }
   else
     {
